### ISSUE TITLE
AdGuard always loads in Dock...?

### ISSUE THREAD

**Issue by Cerberus-Hades** [AdguardTeam/AdGuardForSafari#818]
### Please answer the following questions for yourself before submitting an issue.

- [x] I am running the latest version
- [x] I made sure all content blockers are enabled in Safari settings
- [ ] I checked the documentation and found no answer
- [ ] I checked to make sure that this issue has not already been filed

### AdGuard version

AdGuard für Safari Version 1.11.11 (Apple Version)

### Environment

```markdown
- OS:macOS Ventura 13.2.1 - Safari-Version:16.3 
(Apple Version) AdGuard für Safari Version 1.11.11
```

### What filters do you have enabled?

AdGuard Base filter

### Issue Details

Steps to reproduce:
1. mac OS Ventura boots to 13.2.1
2. AdGuard for Safari starts "always in the Dock" with afari Verison 16.3 - why?
3.AdGuard for Safari used to always start but not in the dock, but now it does?

### Expected Behavior

I expect AdGuard for Safari version 1.11.1
(Apple Version) when booting up the computer, as it used to be active without being loaded in the dock.

### Actual Behavior

AdGuard for Safari version 1.11.1
(Apple version) always ends up in the dock after startup, wasn't that the case before? The protection was active without loading it into the dock.

### Screenshots

<details><summary>Screenshot 1:</summary>

</details>

**Comment by Versty** [comment]
@Cerberus-Hades This is clearly a bug, will be fixed soon! 

Closed as a duplicate of #805.

**Comment by Cerberus-Hades** [comment]
Thanks very much!

**Comment by BJKle** [comment]
@Cerberus-Hades 
Is this issue solved for you? I have Version 1.11.13 and it reappears again in the dock after removing and rebooting. Maybe you could reopen ist. Thank you
